test/issue-2928/test.k

```diff
@@ -18,8 +20,7 @@ module TEST
 
     rule X in SetItem(Y) => false requires X =/=Int Y [simplification]
 
-    rule .Set -Set S => .Set [simplification]
-    rule (SetItem(X) REST) -Set S => REST -Set S requires X in S [simplification]
-    rule (SetItem(X) REST) -Set S => SetItem(X) (REST -Set S) requires notBool (X in S) [simplification, label(toApply)]
-    
+    rule SetItem(X) |Set SetItem(Y) => SetItem(X) SetItem(Y)
```

Comment:
The newer K version does not rewrite `|Set` to `-Set` in symbolic terms, so this test performs a different simplification now. Also, now importing the symbolic set simplifications from `SET-SYMBOLIC` instead of explicitly stating the required ones for the test.